dist/index.html

```diff
@@ -0,0 +1,296 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500 &display=swap" rel="stylesheet">
```

Comment:
```suggestion
<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
```
Багато тестів не пройшли, скоріш за все, через те, що шрифт підключений не зовсім вірно